resources/dicts/patrols/mountainous/med/any.json

```diff
@@ -397,7 +397,7 @@
                 "herbs": ["cobwebs"]
             },
             {
-                "text": "p_l visits a few of their favourite places to gather cobwebs. When they get back to camp, they end up startling several cats on their way back to their den - p_l is practically a walking cobweb, impossible to see beneath the sheer mass they have gathered.",
+                "text": "p_l visits a few of their favorite places to gather cobwebs. When they get back to camp, they end up startling several cats on their way back to their den - p_l is practically a walking cobweb, impossible to see beneath the sheer mass they have gathered.",
```

Comment:
Missing some pronoun tags

Reply:
Almost all of the med patrol are, I'm afraid. I've only pronoun tagged the patrols on this branch that are new to the game, we're still slowly chugging through the jsons pronoun tagging old ones